### ISSUE TITLE
Linkable urls

### ISSUE THREAD

**Issue by daleharvey** [neojski/visualizeRevTree#3]
If you change the form to just visit

```
http://neojski.github.io/visualizeRevTree/?doc=http://daleharvey.iriscouch.com/grocery/1323264524753-66494280-29b7-4b87-9fba-501e14e35420
```

then people can link to visualisations

**Comment by neojski** [comment]
I like this idea very much.

**Comment by daleharvey** [comment]
Sweet, it works :) https://github.com/neojski/visualizeRevTree/pull/6, The url isnt as clean as I mentioned since it needs encoding, but it works well I think

**Comment by neojski** [comment]
Fixed in #8
